tox.ini

```diff
@@ -1,40 +1,73 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
-# Note: if you add new Python versions, please add them to
-# [testenv:coverage] depends as well!
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
-    coverage run -p -m zope.testrunner --test-path=src []
+    zope-testrunner --test-path=src []
 extras =
     test
-deps =
-    coverage
-setenv =
-     ZOPE_INTERFACE_STRICT_IRO=1
-# On certain platforms, at least OS X, pypy3 does not correctly read
-# a LANG environment variable; when cpython3.2 would return 'utf-8'
-# from locale.getpreferredencoding(), pypy3 returns 'us-ascii', causing
-# two test failures. Fix that by setting something it does read.
-     pypy3: LC_CTYPE=en_US.UTF-8
```

Comment:
Does anybody know if this workaround is still necessary?

Do we have any CI on Mac OS?

Do we know any developers who use Mac OS and could weigh in?

Reply:
It might still be necessary. 

```console
$ env | grep "LANG\|LC" | sort
LANG=en_US.UTF-8
LC_ALL=en_US.UTF-8
LC_CTYPE=C
LC_TERMINAL=iTerm2
LC_TERMINAL_VERSION=3.4.3beta1
$ pypy3 --version
Python 3.6.9 (d38cd66c14b8, Sep 23 2020, 08:01:31)
[PyPy 7.3.2 with GCC Apple LLVM 12.0.0 (clang-1200.0.32.2)]
$ pypy3 -c 'import locale; print(locale.getpreferredencoding())'
UTF-8
$ LANG=en_US.UTF-8 LC_ALL= pypy3 -c 'import locale; print(locale.getpreferredencoding())'
US-ASCII
```

Reply:
@jamadden what you show is correct POSIX locale semantics: LC_CTYPE overrides LANG (and LC_ALL overrides LC_CTYPE).  LC_CTYPE=C implies US-ASCII as the locale encoding.

Python 3.7 [decided to ignore locale settings and use UTF-8 when the locale is set to C/POSIX](https://docs.python.org/3/whatsnew/3.7.html#whatsnew37-pep538), but your PyPy3 is a 3.6.

So far it looks to me that this tox.ini hack is a workaround for someone's broken local environment, and doesn't really need to be there?  Or, at least, it has no reason to be PyPy-specific, since CPython < 3.7 will also use US-ASCII in these circumstances?

Reply:
I am running on Mac OS: The tests run fine without this setting.